README.rst

```diff
@@ -1,7 +1,26 @@
+flit
```

Comment:
*Since you seem to dislike the idea of adding the logo to the README in general*:
The reason I've put it there in the first place is because the logo is great for capturing the interest of potential users. Yes, this is a programming tool not an end-user app, but in a lot of ways this tool is also a statement that “Python packaging can be simple”, which the logo IMHO conveys pretty nicely. Finally it's just nice to have a little bit of visual change in the otherwise pretty black-and-white, matter-of-factly documentation that we programmers always stare at. It really adds a lot.
(Remember: We're talking about adding 150px of visual information at the top of a `README` here, not about addind a page-size flicking advertisment.)

> If the README includes the logo as well, the docs would have it twice.

I don't see how having the logo in the README means that the logo cannot be also be visible on the documentation?